.buildkite/scripts/package.sh

```diff
@@ -8,14 +8,7 @@
 set -eo pipefail
 
 TYPE="$1"
-PLATFORM_TYPE=$(uname -m)
 
-PLATFORMS=''
-PACKAGES=''
-if [[ ${PLATFORM_TYPE} == "arm" || ${PLATFORM_TYPE} == "aarch64" ]]; then
-  PLATFORMS='linux/arm64'
-  PACKAGES='docker'
-fi
 
```

Comment:
`PLATFORMS` was defaulted, henve the BK env variables was not working as expected. Avoid using the script but honour the env variables defined in the Buildkite pipeline itself.